R/post.R

```diff
@@ -80,23 +80,7 @@ post_tweet <- function(status = "my first rtweet #rstats",
 
   ## if delete
   if (!is.null(destroy_id)) {
-    ## validate destroy_id
-    stopifnot(is.character(destroy_id) && length(destroy_id) == 1)
-    ## build query
-    query <- sprintf("statuses/destroy/%s", destroy_id)
-    ## make URL
-    url <- make_url(query = query)
-
-    ## send request
-    r <- TWIT(get = FALSE, url, token)
-
-    ## if it didn't work return message
-    if (r$status_code != 200) {
-      return(httr::content(r))
-    }
-    ## if it did, print message and silently return response object
-    message("your tweet has been deleted!")
-    return(invisible(r))
+    return(post_destroy(destroy_id))
```

Comment:
Added the .Deprecated message. However, the documentation of `post_destroy` is  in `post_tweet`. So I haven't deleted the argument documentation. Do you think it is better in its own page?

Reply:
I think you're better off adding a lifecycle dependency since the user experience is much better (i.e. only one warning every 8 hours) unless you have a strong reason not to add a lightweight dependency.

Reply:
I don't see the annoyance of getting a message each time a users use post_tweet to eliminate a tweet too much. They can move to the new function if this bothers them too much. But I'll replace it by the lifecycle version. Regarding user experience, we can also use cli for the message and notifications to users to make the package more user friendly.